eiffel-vocabulary/EiffelActivityFinishedEvent.md

```diff
@@ -0,0 +1,34 @@
+# EiffelActivityFinishedEvent
+The EiffelActivityFinishedEvent declares that a previously queued activity (declared by [EiffelActivityQueuedEvent](./EiffelActivityQueuedEvent.md)) has finished.
+
+## Data Members
+### data.outcome
+__Type:__ Object  
+__Required:__ Yes  
+__Description:__ The outcome of the activity.
+
+#### data.outcome.verdict
+__Type:__ String  
+__Required:__ Yes  
+__Legal values:__ SUCCESS, FAILURE, ERROR, ABORTED, TIMEOUT  
```

Comment:
If we map those to ones that are available in Jenkins, then we could also add UNSTABLE. Which mean activity was able to run until the end, but verification check result is different from expected